boards/sensebox_samd21/Kconfig

```diff
@@ -23,4 +23,7 @@ config BOARD_SENSEBOX_SAMD21
 
     select HAVE_SAUL_GPIO
 
+    select HAVE_MTD_SDCARD_DEFAULT
```

Comment:
Hm, I would say the selection of a module should be done `select MODULE_MTD_SDCARD_DEFAULT` which depends on a feature `HAVE_MTD_SDCARD_DEFAULT`.

Reply:
`drivers/mtd_sdcard/Kconfig` should define the feature `HAVE_` and the `MODULE_`.

Reply:
Hm but why? This is also what `HAVE_SAUL_GPIO` does - why the added indirection?

Reply:
Ok. I just thought we model the dependencies exactly like in `Makefile.dep` where we have
```
ifneq (,$(filter mtd,$(USEMODULE)))
  USEMODULE += mtd_sdcard_default
endif
```
which would be in Kconfig:
```
select MODULE_MTD_SDCARD_DEFAULT if MODULE_MTD
```
If you want just to indicate that the board has a feature `HAVE_MTD_SDCARD_DEFAULT` and the dependencies defined anywhere else pull in the module, I'm fine with it. I just thought you want to model `USEMODULE` dependency with `HAVE_` feature in Kconfig.

Reply:
I'm happy with whatever works at the least complexity.
The

    select MODULE_MTD_SDCARD_DEFAULT if MODULE_MTD

is done in `HAVE_MTD_SDCARD_DEFAULT` so that already works that way.